lib/js/src/React.js

```diff
@@ -3,5 +3,13 @@
 
 var Ref = /* module */[];
 
+function Provider(C) {
```

Comment:
There has got to be a way to eliminate this runtime. I really would love to avoid this.

Reply:
I would love that, too! But I am not aware of any way around this, unfortunately. :-(